astro/src/content/articles/security/adaptive-mfa-user-experience.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Why Adaptive MFA is a Game Changer for User Experience 
+description: Users find traditional multi-factor authentication to be a burden. In this article, we examine how adaptive MFA can help improve the user experience (UX) rather than hinder it.
+author: Matt Keib
+icon: /img/icons/adaptive-mfa.svg
+darkIcon: /img/icons/adaptive-mfa-dark.svg
+section: Security
+---
+
+A commonly held belief is that security measures make accessing resources harder---but only for end users. Attackers excel at finding ways around security tools and measures. They take advantage of annoyed users to be more successful at [phishing campaigns](https://www.barracuda.com/support/glossary/phishing-campaign) and easily collect those long, complex, inconvenient passwords.
```

Comment:
one thing we need to position in this is that adaptive MFA is really focused on workforce for now. There are some consumer use cases but we need to draw the line here between the article's intended audience (more of a security professional or IT person), than our fusionauth audience (app developers)